assets/src/design-system/components/typography/headline/index.js

```diff
@@ -34,10 +34,19 @@ export const Headline = styled.h1`
       preset: theme.typography.presets.headline[size],
       theme,
     })}
+  ${({ as, theme }) =>
+    as === 'a' &&
+    css`
+      :hover {
+        color: ${theme.colors.fg.linkHover};
+      }
+
+      ${themeHelpers.focusableOutlineCSS(theme.colors.border.focus)}
+    `}
 `;
 
 Headline.propTypes = {
-  as: PropTypes.oneOf(['h1', 'h2', 'h3', 'h4', 'h5', 'h6']).isRequired,
+  as: PropTypes.oneOf(['h1', 'h2', 'h3', 'h4', 'h5', 'h6', 'a']).isRequired,
```

Comment:
💭 I think Headline shouldn't be a link. We can always wrap it with a `Link` component, right? This should just be a heading

imo giving it the ability to be an anchor would go against the idea that this element should only be for headings

Reply:
Morten was having the same problem here: https://github.com/google/web-stories-wp/pull/6570#issuecomment-789794664

Reply:
hmm I see. Okay cool 👍